urpautils/file_utils.py

```diff
@@ -105,6 +106,17 @@ def read_txt_file(file_name: str, encoding: str = "utf-8") -> str:
         return txt_file.read()
 
 
+def read_json_file(file_name: str, encoding: str = "utf-8") -> dict:
+    """Reads a json file
+
+    :param file_name:    path
+    :param encoding:     encoding to use
+    :return:             str content
```

Comment:
One last thing - this now should be `:return:       dict content`

After this fix I'm ready to merge your pull request

Reply:
Sorry, I should've changed that too. It's fixed in the 66dc8a3 commit.